docs/developers-guide/devenv.md

```diff
@@ -141,6 +141,34 @@ And of course your Jetty development server is available via
 clojure -M:run
 ```
 
+You can also start a REPL another way (e.g., through your editor) and then call
+
+```
+(require 'dev)
+(dev/start!)
+```
+
+To start the server (at http://localhost:3000/). This will also set up / migrate your application database. To actually
+use Metabase, don't forget to start the frontend as well (e.g. with `yarn build-hot`).
+
+### The application database
+
+By default, Metabase uses H2 for its application database, but we recommend using Postgres. This is configured with
+several properties that can be set as environment variables or in a `deps.edn`. One approach is:
+
+```
+;; ~/.clojure/deps.edn
+
+{:aliases
+ {:user
```

Comment:
also worth highlighting that you can pass a full connection string like
`mb.db.connection.uri=postgres://<user>:<password>@localhost:5432/<dbname>`